### ISSUE TITLE
[Bug] Total count is wrong

### ISSUE THREAD

**Issue by stergiosba** [stergiosba/PaxRL#6]
This calculation is wrong currently. Produces zeros in a very simple scenario
    total_count = neighbors_count+leader_str*leader_neighbors_count

**Comment by stergiosba** [comment]
Produces even -1. Something is very wrong with this line:
in script.py

    neighbors_count = jnp.sum(neighbors_matrix, axis=0)-leader_neighbors_count

**Comment by stergiosba** [comment]
Fixed by using two masks:

    # Performs neighbors masking.
    total_mask = (total_count>0)[:,None]
    neighbors_mask = (neighbors_count>0)[:,None]
